### ISSUE TITLE
Please test updates before release?

### ISSUE THREAD

**Issue by minim8** [mchristopher/PokemonGo-DesktopMap#401]
Hey, I dont want to sound ungrateful - I really appreciate all the work you guys do here and I have no problem donating some coins but while the updates brought some comfortable improvements they also broke the main mechanics of this program. (kind of ironic since it feels like pokemon go all over again - hello 3 step bug). Wouldnt it be simpler to just wait with the updates until you can resolve the main functionalties and at least gurantee stability for at least 2 hours?

To summarize just a few of the current problems (Version 0.1.5):
-Filter options are deleted everytime you log out (incredibly annoying)
-Updating doesnt work properly anymore
-Shows non exisiting Pokemon "2min 45sec bug" (checked 6 times now in real life and with other apps)
-Doesn't show all of the existing Pokemon, even with the slow scan (checked as well)
-Scanning speed is incredibly slow

**Comment by AVellucci** [comment]
I agree with maybe a little more testing before releasing new versions.

The Scan is much slower and very in accurate. running 0.1.3 side by side with 0.1.5 not only is the earlier version quicker in the scan it finds more pokemon and also places them in the correct location. 
0.1.5's "non-existant" pokemon is due to the sprites being upserted to the wrong coordinates.

Fast Scan for 0.1.5 is still not as quick as 0.1.3
and just as accurate as the slow scan, are there really any differences between slow and fast scan in 0.1.5?

**Comment by wowoo123** [comment]
these new .exe versions are not very good, the  .zip were much better, im using version 0.1.3 now

**Comment by AVellucci** [comment]
@wowoo123 Please do not spam these messages throughout issues, they offer no feedback and are otherwise just obtrusive, I'm only saying this because you've posted the same message in 3 issues so far.

**Comment by calgar** [comment]
"Shows non exisiting Pokemon "2min 45sec bug" (checked 6 times now in real life and with other apps)"

Is this one really a bug?   I though they were "lured" pokemon".  That's why I thought they were out of the scanned boxes... I had an alarm on a Snorlax...it triggered.. went there... and there he was next to a lured Pokestop

**Comment by minim8** [comment]
@calgar 
Well I checked it many times and there was neither the Pokemon nor a lure nearby, so from my experience these were really non exisiting to begin with. The app also shows lured Pokestops so with the current version you can easily verify that these bug-Pokemon are shown even without having a lure close by.

On a sidenote: Congrats and lucky you ;) I've been looking for Snorlax for ages without any luck so far.
